packit/constants.py

```diff
@@ -168,3 +168,11 @@
     # modules default to string because Copr stores it as string in the DB
     "additional_modules": "",
 }
+
+PACKAGE_LONG_OPTION = "--package"
+PACKAGE_SHORT_OPTION = "-p"
+PACKAGE_OPTION_HELP = (
+    "Package to {action}, if more than one available, "
+    "like in a monorepo configuration. "
```

Comment:
Add something like: "Use it multiple times to select multiple packages." So that it's clear how to do this.